pyfolio/utils.py

```diff
@@ -402,7 +406,7 @@ def to_series(df):
     return df[df.columns[0]]
 
 
-@deprecated(msg=DEPRECATION_WARNING)
+@deprecated(msg=DATAREADER_DEPRECATION_WARNING)
```

Comment:
These are already going to raise a warning since they call into deprecated empryical functions, right? Do we want to raise these warnings twice?

Reply:
For any of the functions that are just passthroughs to empyrical, I think we can just change our definitions from

```python
def some_func(arg):
    return empyrical.some_func(arg)
```
to just

```python
some_func = empyrical.some_func
```

Reply:
Good point; refactored accordingly.